build-system/pr-check/build-targets.js

```diff
@@ -151,6 +174,12 @@ const targetMatchers = [
       return isOwnersFile(file) || file == 'build-system/tasks/check-owners.js';
     },
   },
+  {
+    targets: ['PACKAGE_UPGRADE'],
+    func: file => {
+      return file == 'package.json' || file == 'yarn.lock';
```

Comment:
Does this catch all `package.json` files?

Reply:
It does not, and that's by design.

All inner directory `package.json` files are covered by their respective build targets. (e.g. changes to `build-system/tasks/e2e/package.json` will trigger the `E2E_TEST` target.)

Reply:
SG